--- stractistics/web_ui.py.org	2010-07-31 00:14:52.000000000 +0200
+++ stractistics/web_ui.py	2010-07-31 00:15:33.000000000 +0200
@@ -100,7 +100,7 @@
         config = util.read_config_options(self.env.config)
         
         #Patch for Trac 0.11
-        if trac.__version__.find('0.11') != -1:
+        if trac.__version__.find('0.11') != -1 or trac.__version__.find('0.12') != -1:
             chrome = Chrome(self.env)
             chrome.populate_hdf(req)
                 
@@ -124,4 +124,4 @@
         return template, content_type
 
     
-                
\ No newline at end of file
+                
