--- freevo.org	2010-01-13 12:55:55.000000000 +0100
+++ freevo	2010-01-13 13:18:40.000000000 +0100
@@ -179,8 +179,9 @@
     """
     _debug_('version=%r' % (sys.version))
     if sys.hexversion >= 0x02040000:
-        # python seems to be ok
-        search = ('python', 'python2')
+        # this python seems to be ok, so try that first, else just use python
+        exe = sys.executable
+        search = (exe, 'python', 'python2')
     elif sys.hexversion >= 0x02030000:
         # try python2.4, else take python
         search = ('python2.4', 'python')
